### PR TITLE
Add permissions to crdt requests.

### PR DESCRIPTION
**Pull request by seancribbs**
@Vagabond @russelldb

### CODE REVIEW

File: src/riak_kv_pb_crdt.erl [hunk 1]
```diff
@@ -214,3 +215,8 @@ get_context(Ctx, true) ->
 mods_match(BucketMod, OpType) ->
     OpMod = riak_kv_crdt:to_mod(OpType),
     OpMod == BucketMod.
+
+permission_for(#dtupdatereq{bucket=B, type=T}) ->
```
Comment (Vagabond) [inline src/riak_kv_pb_crdt.erl:348]: Will type always be a binary here? Never undefined?

Comment (seancribbs) [inline src/riak_kv_pb_crdt.erl:348]: `type` is `required bytes` in the protobuffs message.

**Comment by russelldb** [comment]
Looks good to me. And wow. Is that it? Bravo @Vagabond for making it so neat.

**Comment by Vagabond** [comment]
Yes, this is all you need. +1 from me, although a test would be nice (maybe added to the pb_security test in the security riak_test PR:  basho/riak_test#385 )

**Comment by seancribbs** [comment]
@Vagabond I'll add to basho/riak_test#385.

@russelldb We'll have to add them to HTTP when we get around to solidifying that API.

**Comment by seancribbs** [comment]
I've rebased this so it applies cleanly. Still need a test in pb_security, I suppose.

**Comment by seancribbs** [comment]
^^ @lenary pb_security and http_security need additions if you have time. If not, I'll add them next week.

**Comment by russelldb** [comment]
test passes now, are you going to get this merged too?

**Comment by seancribbs** [comment]
yup yup

**Pull request merged**